Tests/TestingApps/PurchaseTesterSwiftUI/Core/ConfiguredPurchases.swift

```diff
@@ -7,6 +7,7 @@
 
 import Foundation
 
+#if DEBUG
```

Comment:
Oops, remove please

Reply:
I had to put back the `@testable import` since `DangerousSettings` initializer that let's you configure `internalSettings` is marked as... internal 😅